### DISCUSSION TITLE
Possible error: Chapter 3, Problem 7

### DISCUSSION THREAD

**Discussion by bryzhang** [fifthist/Introduction-To-Probability-Blitzstein-Solutions#28] [General]
I think the product function should account for the probability that Bob not only makes it to a given level, but fails to pass to the next level, and it doesn't look like it's currently doing that. So:

$P(X = 1) = (1 - p_1)$
$P(X = k) = p_1 \ldots p_{k - 1} * (1 - p_k)$ for $1 < k < 7$
$P(X = 7) = p_1 \ldots p_6$

@JoeZiminski Tagging you in case you're interested in taking a look.

**Comment by JoeZiminski** [comment]
I agree, I got the same solution. 👍
